chandra_aca/maude_decom.py

```diff
@@ -428,7 +428,7 @@ def _aca_header_2(bits):
         -------
         dict
         """
-        bits = _unpack("BbbbbBB", bits)
+        bits = _unpack("BBbbbBB", bits)
```

Comment:
I'm wondering if it should be:
```
BBbbbbB
0123456
```
Below we see that `bits[2], [3], [4], and [5]` are all `TEMP*` variables, so I would be slightly surprised if `TEMPSEC` is unsigned while all the rest are signed. But maybe that is the way it is?

Off topic, but being more used to type annotations now, it seems like changing `bits` from `bytes[7]` to `tuple[int]` is pretty confusing. Going forward making a new variable like `values` would be a good thing. I struggled at first to understand the code because `bits` changed from what is in the docstring and it also doesn't match the pattern for the other two header format functions.

Reply:
That's a good catch. That temperature is not in any test, so it was not checked, and we never use it. I just fixed it.

I also renamed the intermediate variable